frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { checkErrors } from '../../support';
+import { isLoaded } from '../../views/common';
+import { masthead } from '../../views/masthead';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('pseudolocalizes masthead', () => {
+    cy.log('test masthead');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    masthead.clickMastheadLink('help-dropdown-toggle');
+    cy.byTestID('application-launcher-item', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes navigation', () => {
+    cy.log('test navigation');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    cy.byTestID('nav', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes activity card', () => {
+    cy.log('test activity card components');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    cy.byTestID('activity', /\[[^a-zA-Z]+\]/);
+    cy.byTestID('activity-recent-title', /\[[^a-zA-Z]+\]/);
+    cy.byTestID('ongoing-title', /\[[^a-zA-Z]+\]/);
+    cy.byTestID('events-view-all-link', /\[[^a-zA-Z]+\]/);
+    cy.byTestID('events-pause-button', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes utilization card', () => {
+    cy.log('test utilization card components');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    cy.byTestID('utilization-card-item-text', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes timestamps', () => {
+    cy.log('test timestamps');
+    cy.visit('/k8s/all-namespaces/events?pseudolocalization=true&lng=en');
+    isLoaded('timestamp');
+    cy.byTestID('timestamp', /\[[^a-zA-Z]+\]/);
```

Comment:
Not sure you need the `isLoaded`.   `cy.byTestID('timestamp', /\[[^a-zA-Z]+\]/)` will fail if the elem doesn't exist.

Reply:
Seems happy enough without it!